scss/inc/_functions.scss

```diff
@@ -270,3 +270,13 @@ Usage:
 @mixin disableSelect() {
     @include vendor-prefix(user-select, none, property);
 }
+
+/* based on "visually-hidden" mixin in LDS for accessibility goals */
+@mixin visuallyHidden() {
```

Comment:
I think `position: absolute;` is still required, to remove it from the flow (so that neighbouring elements would not be moved by 1px). We have it in the LDS, and the [original article](https://snook.ca/archives/html_and_css/hiding-content-for-accessibility) has it too.

Reply:
Agree. Very strange, but i copied this with position property. In any case i'll fix this.